### PR TITLE
Typos and hides dotflies

### PR DESCRIPTION
**Pull request by FabioAntunes**

### CODE REVIEW

File: kubernetes-101/deployments/step4.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ You should see two **ReplicaSets** why? Our deployment behind the scenes has cha
 
 Let's check if our deployements was succesfull:
 
-`kubectl rollout status deploymet/nginx-deployment`{{execute}}
+`kubectl rollout status deployment/nginx-deployment`{{execute}}
 
 You should also see that we have a new `rollout` history:
 
-`kubectl rollout status deploymet/nginx-deployment`{{execute}
+`kubectl rollout status deployment/nginx-deployment`{{execute}
```
Comment (Jenniferstrej) [inline kubernetes-101/deployments/step4.md]: another curly brace here {{execute}

**Review by Jenniferstrej — CHANGES_REQUESTED** [review]

File: kubernetes-101/deployments/step4.md (hunk 1 above)
Comment (olikeys) [inline kubernetes-101/deployments/step4.md]:
```suggestion
`kubectl rollout status deployment/nginx-deployment`{{execute}}
```

Comment (olikeys) [reply kubernetes-101/deployments/step4.md]: because I love suggestions :D

Comment (FabioAntunes) [reply kubernetes-101/deployments/step4.md]: sorry Jen @olikeys made it easier

**Review by Jenniferstrej — APPROVED** [review]

**Pull request merged**